lightkurve/search.py

```diff
@@ -438,20 +433,26 @@ def _search_products(target, radius=None, filetype="Lightcurve", cadence='long',
     -------
     SearchResult : :class:`SearchResult` object.
     """
-    observations = _query_mast(target, cadence='long', radius=radius)
+    observations = _query_mast(target, project=project, cadence='long', radius=radius)
+
     if len(observations) == 0:
         raise SearchError('No data found for target "{}."'.format(target))
-    products = Observations.get_product_list(observations)
-    result = join(products, observations, join_type='left')  # will join on obs_id
+
+    # mask out FFIs from observations    
+    mask = np.array(['FFI' not in obs['target_name'] and
+                     'FFI' not in obs['obs_collection'] for obs in observations])
```

Comment:
Do you think this hard-coded mask is necessary?  I imagine the call to `_filter_products` on line 449 below will take care of retaining only the `filetype` requested?

Reply:
This was added because the `join` on line 446 fails when FFIs are included due to missing keywords in their table entries. I struggled with the simplest way to exclude them for both Kepler and TESS, but if you see a clearer fix let me know!